build/Targets/RepoToolset/VisualStudio.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <VsixVersion Condition="'$(VersionSuffixDateStamp)' != ''">$(VersionPrefix).$(VersionSuffixDateStamp)$(VersionSuffixBuildOfTheDayPadded)</VsixVersion>
+    <VsixVersion Condition="'$(VersionSuffixDateStamp)' == ''">42.42.42.9999999</VsixVersion>
```

Comment:
❓ Why not 42.42.42.42424?

Reply:
It could be whatever. The last part of the VSIX version is normally 7 digits and I wanted it to be consistent. I guess it could be 4242424